docs/exercises/demography/index.qmd

```diff
@@ -27,50 +31,317 @@ to prepare your working directory.
 
 :::{.panel-tabset}
 
-#### UPPMAX
+#### {{< fa server >}} UPPMAX
+
+<!-- markdownlint-disable MD038 -->
+
+Move to your course working directory `/proj/{{< var uppmaxproject
+>}}/users/USERNAME`, create an exercise directory called
+`demography` and `cd` to it:
+
+<!-- markdownlint-enable MD038 -->
+
+```bash
+cd /proj/{{< var uppmaxproject >}}/users/USERNAME
+mkdir -p demography
+cd demography
+```
+
+You will run most analyses on Rackham, so just go to your directory to start. You will need to download the results files to you computer afterwards, you can use `wget` like you'd done multiple times by now.
 
-#### Local
+
+#### {{< fa laptop >}} Local
+
+You will run R, either via `R Studio` or on the command line, you can choose.
+
+I recommend downloading the `Rmd` file that has all the instructions for this exercise and all the R code necessary to generate the plots we want to see today.
+
+Just download it from `{{< var webexport.url
+>}}/demography/demography.Rmd` with `wget`^[The password is provided
+by the course instructor]:
+
+```bash
+wget --user pgip --password PASSWORD {{< var webexport.url >}}/demography/demography.Rmd
+```
 
 :::
 
 :::
 
 ::: {.callout-note collapse=true}
 
-## Tools
+:::
 
-:::{.panel-tabset}
+:::
 
-#### Listing
+::: {.callout-note collapse=true}
 
-- [psmc](https://github.com/lh3/psmc) [@li_InferenceHumanPopulation_2011]
+## Tools
+
+:::{.panel-tabset}
 
 #### UPPMAX modules
 
-Execute the following command to load modules:
+Everything you need is on Uppmax. Execute the following command to load modules:
 
 ```{bash }
 #| label: uppmax-load-modules
 #| echo: true
 #| eval: false
-module load uppmax bioinfo-tools \
-    psmc/0.6.5-r67-e5f7df5
+module load bioinfo-tools samtools PCAngsd ANGSD bcftools plink htslib
 ```
 
-#### Conda
 
-Copy the contents to a file `environment.yml` and install packages
-with `mamba env update -f environment.yml`.
+:::
+
+:::
+
+
+
+## Practical information
+
+This exercise is divided into two parts. The first part will happen on Rackham. All the files you need are located here: ```/proj/naiss2023-22-1084/private/demography```. The second part will happen inside R Studio locally.
```

Comment:
Also here you can use the shortcode {{< var uppmaxproject >}} instead of hardcoding

Reply:
Done!